cypress/e2e/user.cy.js

```diff
@@ -1,12 +1,24 @@
 /// <reference types='cypress' />
 /// <reference types='../support' />
 
-describe('User', () => {
-  before(() => {
+const user = {
+  username: 'wombat098',
+  email: 'wombat098@i.ua',
+  password: '1234567890'
+};
 
+describe('User', () => {
+  beforeEach(() => {
+    cy.login(user);
```

Comment:
Ensure that the `cy.login(user)` command is correctly implemented in your support files. This command should handle the login process for the user object provided.